cupyx/scipy/ndimage/interpolation.py

```diff
@@ -309,7 +310,8 @@ def map_coordinates(input, coordinates, output=None, order=3,
 
 
 def affine_transform(input, matrix, offset=0.0, output_shape=None, output=None,
-                     order=3, mode='constant', cval=0.0, prefilter=True):
+                     order=3, mode='constant', cval=0.0, prefilter=True,
```

Comment:
let;s mark this as purely kwarg
```suggestion
                     order=3, mode='constant', cval=0.0, prefilter=True, *,
```